src/components/products/product-subscriptions/react/ProductSubscriptionsTable.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import {
+    Body1Strong,
+    Spinner,
+    Table,
+    TableBody,
+    TableCell,
+    TableHeader,
+    TableHeaderCell,
+    TableRow,
+} from "@fluentui/react-components";
+import { Subscription } from "../../../../models/subscription";
+
+export const ProductSubscriptionsTable = ({
+    subscriptions,
+    working,
+}: {
+    subscriptions: Subscription[];
+    working: boolean;
+}) =>
+    working ? (
+        <Spinner
+            label="Loading Subsriptions"
+            labelPosition="below"
+            size="small"
+        />
+    ) : !subscriptions || subscriptions.length === 0 ? (
+        <Body1Strong>You don't have subscriptions yet.</Body1Strong>
+    ) : (
+        <div className={"fui-table"}>
```

Comment:
I believe we can get rid of this wrapper and apply fui-table class straight to the table.